src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverted = {};
+
+  for (const prop in robot) {
+    if (inverted[robot[prop]] !== undefined) {
```

Comment:
this should work the same way if we just convert it to boolean value this way. It seems to me you need just to check whether this property is not empty, not to check whether it equals undefined
```suggestion
    if (inverted[robot[prop]]) {
```